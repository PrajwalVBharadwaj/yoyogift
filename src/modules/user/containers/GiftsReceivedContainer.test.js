import React from "react";
import { shallow } from "../../../Enzyme";
import giftsReceivedContainer, {
  GiftsReceivedContainer
} from "./GiftsReceivedContainer";

describe("Testing GiftReceivedContainer", () => {
  it("Snapsot testing", () => {
    const wrapper = shallow(<giftsReceivedContainer isLoggedin={true} />);
    expect(wrapper).toMatchSnapshot();
  });
});
