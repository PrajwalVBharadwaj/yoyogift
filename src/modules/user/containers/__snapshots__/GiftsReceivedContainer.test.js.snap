// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing GiftReceivedContainer Snapsot testing 1`] = `ShallowWrapper {}`;
